README.md

```diff
@@ -120,6 +120,30 @@ Build Options
 | WITH_MAINTAINER_WARNINGS |                          | Build with project maintainer warnings                                                | OFF     |
 | WITH_CODE_COVERAGE       |                          | Enable code coverage reporting                                                        | OFF     |
 
+Build Requirements
+------------------
+
+Supported build systems:
+
+* CMake 3.5.1
+* Configure script (bash/gmake)
+* NMake
+
+Supported C99 compatible compilers:
```

Comment:
This wording makes it seem like no other compilers except the ones listed will work.

Reply:
The build scripts might have limited support for other compilers, but we can't really guarantee the compilation finishes successfully or that the resulting binaries work as expected.